docs/getting_started/basic/zero-shot-prediction/zero-shot-classification.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>Given that we just ran clip - can you explain why you want to run this one as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>